.env.example

```
@@ -5,3 +5,17 @@ DB_NAME=init
 
 # Spring Profile
 SPRING_PROFILE=local
+
+# MinIO (local only)
+MINIO_ROOT_USER=minioadmin
+MINIO_ROOT_PASSWORD=minioadmin
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MINIO_ROOT_PASSWORD key should go before the MINIO_ROOT_USER key

(UnorderedKey)